woocommerce/woocommerce
# New image empty state for variations

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #47788



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Make sure the `Try the new product editor (Beta)` is enabled under `/wp-admin/admin.php?page=wc-settings&tab=advanced&section=features`
2. Go to Products > Add new
3. Add a Name in the General tab
4. Add at least one variation in the Variations tab
5. The click edit one generated variation
6. From the variation edit form and in the General tab, inside the Image section when there is not image set to the current variation the empty state should shown only one product image and the the message should say `Add an image which displays the unique characteristics of this variation.`
![image](https://github.com/woocommerce/woocommerce/assets/13334210/7e8a53a4-6e2a-49a8-bc82-3c01484996e6)



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
